package.json

```diff
@@ -14,7 +14,7 @@
     "build:cart": "yarn --cwd feature-libs/cart run build:schematics && ng build cart --prod",
     "build:cdc": "yarn --cwd integration-libs/cdc run build:schematics && ng build cdc --prod",
     "build:cds": "yarn --cwd integration-libs/cds run build:schematics && ng build cds --prod",
-    "build:libs": "yarn ngcc:lock:remove && ng build core --prod && ng build storefrontlib --prod && yarn build:schematics && yarn build:user && yarn build:asm && concurrently --kill-others-on-fail -m 4 yarn:build:organization yarn:build:product yarn:build:product-configurator yarn:build:storefinder yarn:build:smartedit yarn:build:qualtrics yarn:build:cart yarn:build:assets yarn:build:incubator yarn:build:cdc yarn:build:cds yarn:build:setup yarn:build:tracking",
+    "build:libs": "yarn ngcc:lock:remove && ng build core --prod && ng build storefrontlib --prod && yarn build:schematics && yarn build:user && yarn build:asm && yarn:build:tracking && concurrently --kill-others-on-fail -m 4 yarn:build:organization yarn:build:product yarn:build:product-configurator yarn:build:storefinder yarn:build:smartedit yarn:build:qualtrics yarn:build:cart yarn:build:assets yarn:build:incubator yarn:build:cdc yarn:build:cds yarn:build:setup",
```

Comment:
should be yarn build:asm && yarn build:tracking

currently, it's set to yarn:build:tracking <--- remove the `:` and put a space